admin/requirements.py

```diff
@@ -14,7 +14,7 @@
 from twisted.python.usage import Options, UsageError
 from twisted.python.filepath import FilePath
 
-MANYLINUX_IMAGE = "quay.io/pypa/manylinux1_x86_64:latest"
+ALPINE_IMAGE = "alpine:latest"
```

Comment:
i am always skeptical about using `:latest` for the reason that src code will randomly change to use a new version of a container if it comes out on quay/dockerhub. `latest` oftem means the `most recently pushed`. So as long as we know this shouldn't introduce any issues im OK with it, but figured i'd put the point here.